src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -356,6 +358,19 @@ private static Job CreateJobForGivenRuntime(Job baseJob, string runtimeId, Comma
                     builder.TargetFrameworkMoniker(runtime.MsBuildMoniker);
 
                     return baseJob.WithRuntime(runtime).WithToolchain(builder.ToToolchain());
+                case RuntimeMoniker.Wasm:
+                        IToolchain toolChain = new WasmToolChain("Wasm",
+                                                                 "net5.0",
+                                                                 options.CliPath.FullName,
+                                                                 null,
+                                                                 null,
+                                                                 options.WasmRuntimePackPath,
+                                                                 options.WasmAppBuilderPath,
+                                                                 options.WasmMainJS,
+                                                                 timeOut ?? NetCoreAppSettings.DefaultBuildTimeout);
```

Comment:
it would be great if you could use named arguments here as there are many of them and it would help us avoid bugs